QuentinTheFarmerGame/EventHandlers/MoveEntity/MoveEntityHandler.cpp

```diff
@@ -21,20 +21,22 @@ bool MoveEntityHandler::processEvent(std::shared_ptr<IEvent> event,
 {
     try {
 
-        if (event->getValues().size() != 3)
+        if (event->getValues().size() != 4)
             throw MoveEntityHandlerError("Event values corrupted.");
 
         int indexEntity = std::any_cast<int>(event->getValues()[0]);
         float posX = std::any_cast<float>(event->getValues()[1]);
         float posY = std::any_cast<float>(event->getValues()[2]);
+        float posZ = std::any_cast<float>(event->getValues()[3]);
 
         std::vector<std::pair<int, std::variant<int, float, std::string, bool>>> args;
 
         args.push_back(std::make_pair(ABINetwork::Type::Int, static_cast<int>(indexEntity)));
         args.push_back(std::make_pair(ABINetwork::Type::Float, posX));
         args.push_back(std::make_pair(ABINetwork::Type::Float, posY));
+        args.push_back(std::make_pair(ABINetwork::Type::Float, posZ));
 
-        ABINetwork::sendUpdateComponent(networkUnit, "Position2DComponent", 3, args);
+        //ABINetwork::sendUpdateComponent(networkUnit, "Position2DComponent", 3, args);
```

Comment:
Why did you comment this line ?
If you consider this line useless because I mean you don't want a second network game, please remove.

Reply:
I commented it to be sure that the update doesn't occur, but if we want to update it with the server, I'll uncomment it and modify the args.

Reply:
OK :+1: